### DISCUSSION TITLE
端口转发跟端口只允许指定ip访问的功能

### DISCUSSION THREAD

**Discussion by MetaQuest** [midoks/mdserver-web#677] [Ideas]
你好 今天才发现有这开源的面板真是太好用了  就是有个问题  我想问下面板目前端口转发跟端口只允许指定ip访问的功能是不是还没有呢

**Comment by midoks** [comment]
是什么端口转发。 用防火墙？

**Reply by MetaQuest** [reply]
是的  防火墙的端口转发  还有防火墙只允许指定ip访问访问某个端口
